yellowbrick/features/missing.py

```diff
@@ -0,0 +1,129 @@
+# yellowbrick.features.missing
+# Feature importance visualizer
+#
+# Author:  Nathan Danielsen <nathan.danielsen@gmail.com>
+# Created: Fri Mar 24 8:17:36 2018 -0500
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: missing.py [] nathan.danielsen@gmail.com.com $
+
+"""
+Implementation of missing values visualizers
+
+To Include:
+- Bar
+- Density Matrix (by time, specifiable index)
+- Heatmap
+
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from yellowbrick.utils import is_dataframe
+from yellowbrick.utils import is_structured_array
+from yellowbrick.features.base import DataVisualizer
+
+# from yellowbrick.style.colors import resolve_colors
+
+
+
+##########################################################################
+## Feature Visualizer
+##########################################################################
+
+
+class MissingValuesBarVisualizer(DataVisualizer):
```

Comment:
I've been thinking that naming things with a `Visualizer` or `Viz` suffix stutters a bit, I think that `MissingValuesBar` is just as informative and is a bit easier to handle for users.